### DISCUSSION TITLE
shared_buffers should allocate more RAM

### DISCUSSION THREAD

**Discussion by collimarco** [le0pard/pgtune#52] [General]
Hello,

I used your settings in production with 32GB of RAM and 16 cores. The database has tens of millions of rows. I noticed much better performance by increasing shared buffers from 8 GB (what you suggest) to 12 GB.

Many heavy queries that used to time out, now work perfectly.

Maybe you can consider to increase that value.

**Comment by le0pard** [comment]
> It isn't a silver bullet for the optimization settings of PostgreSQL. Many settings depend not only on the hardware configuration, but also on the size of the database, the number of clients and the complexity of queries, so that optimally configure the database can only be given all these parameters.

From https://pgtune.leopard.in.ua/#/about

Pgtune know nothing about size of your database and tables, complexity of a queries. So it is starting point to tune your database, not final answer for all cases.
